Update VTOL Guide (#198)
R2Northstar/NorthstarWiki: docs/installing-northstar/northstar-installers/vtol-guide.md

# VTOL Guide

VTOL is a mod manager for installing and managing mods for the Northstar client made for Titanfall 2.

> VTOL works on Windows only, and you can find links to download it on [Northstar's website](https://northstar.tf) or [VTOL's GitHub repo](https://github.com/R2NorthstarTools/VTOL). Installing it from the website automatically installs the latest version that will automatically update itself. On the GithHubn repo, you can download the newest version by clicking one of two optionjs. Note that the portable is just a `.exe` file, and not the installer, and as such it won't automatically update itself.

Reviewer: Newlines in source for better diff + fixed typos ^^
```suggestion
VTOL works on Windows only, and you can find links to download it on [Northstar's website](https://northstar.tf) or [VTOL's GitHub repo](https://github.com/R2NorthstarTools/VTOL).
Installing it from the website automatically installs the latest version that will automatically update itself.
On the GithHub repo, you can download the newest version by clicking one of two options.
Note that the portable is just a `.exe` file, and not the installer, and as such it won't automatically update itself.
```

![VTOL installer download (on GitHub, automatically updates](../../images/vtol-download-github.png)

> ![VTOL portable download (on GitHub, doesn't automnatically update](../../images/vtol-download-github-portable.png)

Reviewer: ```suggestion
![VTOL portable download (on GitHub, doesn't automatically update](../../images/vtol-download-github-portable.png)
```

# VTOL Guide

VTOL is a mod manager for installing and managing mods for the Northstar client made for Titanfall 2.

> VTOL works on Windows only, and you can find links to download it on [Northstar's website](https://northstar.tf) or [VTOL's GitHub repo](https://github.com/R2NorthstarTools/VTOL).

Reviewer: ```suggestion
VTOL works on Windows only and you can find links to download it on [Northstar's website](https://northstar.tf) or [VTOL's GitHub repo](https://github.com/R2NorthstarTools/VTOL).
```

> Installing it from the website automatically installs the latest version that will automatically update itself. On the GithHubn repo, you can download the newest version by clicking one of two options. 

Reviewer: Typo
```suggestion
Installing it from the website automatically installs the latest version that will automatically update itself. On the GitHub repo, you can download the newest version by clicking one of two options. 
```

> Note that the portable is just a `.exe` file, and not the installer, and as such it won't automatically update itself.

Reviewer: ```suggestion
Note that the portable is just a `.exe` file and not the installer, as such it won't automatically update itself.
```

![VTOL installer download (on GitHub, automatically updates](../../images/vtol-download-github.png)

![VTOL portable download (on GitHub, doesn't automatically update](../../images/vtol-download-github-portable.png)

## Main Menu

![VTOL Main Window](../../images/vtol-main-window.png)

**The _Main Menu_ of VTOL is where you'll probably spend most of your time.**

This menu has a lot of useful information, such as the version of VTOL you have, the version of Northstar you have, if the Master Server is online, whether EA App/Origin are currently open, and a place to set your game path. 

## Version Numbers

> You can see what version of VTOL/Northstar you have in the top left corner of the VTOL window.

Reviewer: Which one is it, VTOL or Northstar? :P
```suggestion
In the top left corner of the VTOL window you can see what versions of VTOL and Northstar you have installed.
```

Something you might have to do manually is locate your Titanfall 2 install. You can do this by pressing the button of the same name on VTOL, and navigating to your Titanfall 2 directory (If you're unsure of what your game path is, check out the [Default Directories](../troubleshooting.md#game-location) section)

After doing this, Northstar should automatically install. If not, the button at the bottom will read _"Install Northstar"_. 

> Let VTOL install Northstar, and you should be good to go. Once installed, this button will turn into the _"Re-Install Northstar"_ button shown in the screenshot earlier, which you can use to forcibly reinstall Northstar in the case something goes wrong with your Northstar install.

Reviewer: ```suggestion
Let VTOL install Northstar and you should be good to go. Once installed, this button will turn into the _"Re-Install Northstar"_ button shown in the screenshot earlier, which you can use to forcibly reinstall Northstar in the case something goes wrong with your Northstar install.
```

VTOL has a built in-browser for searching [Northstar's Thunderstore page](https://northstar.thunderstore.io/) (the website where most Northstar mods get uploaded to), located at the second tab _"Browse"_ on the tab list. 
You will see the newest uploaded mods here by default, which you can change by applying filters (Note: The sorting is by default Low-High for filters. 
You can change this by pressing the button with an arrow next to the filters button, making it High-Low)

VTOL also has support where all you need to do is drag and drop a mod downloaded from the Thunderstore page onto the _"Mods"_ tab (the second tab on the left side) to automatically install the mod.

## Settings

![VTOL Settings](../../images/vtol-settings-tab.png)

On the way bottom of the tab list, there is a button to open settings for the VTOL manager. You will want to keep most of these on default, but change them as you wish.
It is normal for the _"Restart_As_Admin"_ button to appear untoggled. This is also where you can tick _"Enable_EA_APP_Usage"_ to make the client work better with the EA app, assuming you have it and not Origin.

## Skins

![Skins installation tab](../../images/vtol-skins.png)

> The third settings tab on VTOL is the _"Skins"_ tab. VTOL has support for DDS format skins, which are commonly used for Vanilla Titanfall 2, with Northstar generally opting to convert them to mods.

Reviewer: I would still mention that DDS skins are deprecated.

Like you can use them for vanilla but you have to overwrite files. You could also just use Northstar on vanilla to use the new skin format :P

However, you can still install skins like this for use with Northstar. All you need to do is drag the folder of a DDS format skin onto the Skins screen of VTOL to install it automatically.

## Server

![Launch arguments/dedicated server tab](../../images/vtol-server.png)

The fourth tab on VTOL is the _"Server"_ tab. This section allows you to add Launch Arguments to your client or dedicated server.

> Note that this adds Launch Arguments to change what Northstar does when it launches, not to launch Northstar itself.

Reviewer: The second part of the sentence actually kinda confused me more.

It's just standard launch arg stuff, right?

You can change this by pressing the button with an arrow next to the filters button, making it High-Low)

VTOL also has support where all you need to do is drag and drop a mod downloaded from the Thunderstore page onto the _"Mods"_ tab (the second tab on the left side) to automatically install the mod.

## Settings

![VTOL Settings](../../images/vtol-settings-tab.png)

On the way bottom of the tab list, there is a button to open settings for the VTOL manager. You will want to keep most of these on default, but change them as you wish.
It is normal for the _"Restart_As_Admin"_ button to appear untoggled. This is also where you can tick _"Enable_EA_APP_Usage"_ to make the client work better with the EA app, assuming you have it and not Origin.

## Skins

![Skins installation tab](../../images/vtol-skins.png)

The third settings tab on VTOL is the _"Skins"_ tab. VTOL has support for DDS format skins (the skins with the numbered folders in them), which are commonly used for Vanilla Titanfall 2, with Northstar generally opting to convert them to mods.

> This format is older, and less consistent than converting them to a mod to be used with Northstar (see [Advocate](./#advocate)), and can still be used on Vanilla by making use of a [vanilla profile](../using-northstar/advanced.md#vanilla-profile).

Reviewer: ```suggestion
This format is older and less consistent than converting them to a mod to be used with Northstar (see [Advocate](./#advocate)). It can still be used on Vanilla by making use of a [vanilla profile](../using-northstar/advanced.md#vanilla-profile).
```

![VTOL's Skin Tool implementation](../../images/vtol-skintool.png)

The second screen is an area where you can drag and drop images to the different maps of a skin to make your own skin. This is an implementaion of the [Skin Tool](https://github.com/zxcPandora/Titanfall2-SkinTool).

### Advocate

![VTOL's Advocate implementation](../../images/vtol-advocate.png)

The third screen is an implementation of Spoon's [Advocate Tool](https://github.com/ASpoonPlaysGames/Advocate), where you can convert a DDS formatted skin to a mod that can be installed by everyone normally without the need for the [Skin Tool](https://github.com/zxcPandora/Titanfall2-SkinTool).

> Please give credit to the original skin owner while doing this. (If confused on how to use this, follow the earlier link to Github describes how to set up Advocate. Do note, this version of Advocate isn't perfectly up to date)

Reviewer: This is kinda confusing. There's only a need to give credit when you actually share the skin. You don''t need to do anything if you just use it for yourself.

This kinda makes it sounds like you have to say "thank you skin creator" everytime you launch the game xd

![VTOL's Profiles menu](../../images/vtol-profiles.png)

The sixth tab is the _"Profiles"_ tab, allowing you to create profiles for Northstar. Profiles are seperate Northstar installs, each with their own set of mods and seperated by folders. 

> For example, you can have one profile with your normal set of mods, and another profile for only using core mods (those that common with Northstar/the ones that start with `Northstar.`).

Reviewer: ```suggestion
For example, you can have one profile with your normal set of mods and another profile for only using core mods (those that common with Northstar/the ones that start with `Northstar.`).
```
